src/cable/handler.cr

```diff
@@ -14,7 +14,9 @@ module Cable
       path = context.request.path
       Cable::Logger.info "Started GET \"#{path}\" [WebSocket] for #{remote_address} at #{Time.utc.to_s}"
 
-      context.response.headers["Sec-WebSocket-Protocol"] = "actioncable-v1-json"
+      unless ENV["DISABLE_SEC_WEBSOCKET_PROTOCOL_HEADER"]?
+        context.response.headers["Sec-WebSocket-Protocol"] = "actioncable-v1-json"
+      end
```

Comment:
Since we already include Habitat, what do you think about just making this a Habitat config?

Reply:
Yeah absolutely, that makes sense.